tgui/packages/tgui/components/Chart.tsx

```diff
@@ -50,13 +50,11 @@ const normalizeData = (
     max[1] = rangeY[1];
   }
 
-  const normalized = map((point: Point) => {
```

Comment:
This variable name was self-commenting, now it's hard to say what this code block does.

Reply:
the function is normalizeData :P I would hope that it's normalizing it